Makefile

```diff
@@ -214,8 +214,19 @@ dev/syncdeps:
 
 .PHONY: release/versionbump
 release/versionbump:
+	bazel run //hack/versionbump:versionbump -- patch $(VERSION) > $(PWD)/version.txt
+	# Reload make so it rereads version.txt
+	$(MAKE) release/gen-files
+
+.PHONY: release/gen-files
+release/gen-files:
```

Comment:
> Is there a case where we wouldn't want to be generating specifically for `cockroachdb/cockroach-operator` master? 

I'm not sure... In that case we just don't call this target? :)

> Does it make sense to put a check in to verify that the current repo has all of the latest changes?

Can you elaborate what you mean by "has all of the latest changes" and how to determine that?

Reply:
What I'm trying to say poorly is that because this is driven off a developers laptop, should we make checks to enforce that they have the latest version of master on their local repo? Ensure that there isn't a missing rebase or something so that we can avoid errors. Is that a valid concern?

Reply:
Oh, I see what you mean. Right now this is guarded by GitHub - we cannot push to the master branch directly, you have to go through the PR process in order to merge something to master.

After it's automated and moved to TeamCity or GitHub actions, this is something we get for free by the checkout step. There is a risk of a race condition, but in that case the push step would fail.